### DISCUSSION TITLE
What is the difference between local_mount_point and remote_mount_point?

### DISCUSSION THREAD

**Discussion by ZhiyuanChen** [microsoft/nni#4192] [Q&A]
If I have two azure blob, 'blob1' and 'blob2', both under storage_account 'account', and I would like to mount them at '/mnt/account/blob1' and '/mnt/account/blob2', how should I mount them?

**Comment by acured** [comment]
You can set it at containerName https://nni.readthedocs.io/zh/stable/reference/experiment_config.html?highlight=sharedstorage#containername.

local_mount_point: The path that the storage has been or will be mounted in the local machine.
remote_mount_point(remote mode): The path that the storage will be mounted in the remote machine.

More refs: https://nni.readthedocs.io/en/stable/reference/experiment_config.html?highlight=sharedstorage#sharedstorageconfig
